package.json

```diff
@@ -11,7 +11,7 @@
     "test-e2e": "cross-env NODE_ENV=test BABEL_DISABLE_CACHE=1 mocha --retries 2 --compilers js:babel-register --require ./test/setup.js ./test/e2e.js",
     "lint": "eslint --cache --format=node_modules/eslint-formatter-pretty .",
     "lint-fix": "npm run lint -- --fix",
-    "hot-server": "cross-env NODE_ENV=development node --max_old_space_size=2096 -r babel-register server.js",
+    "hot-server": "cross-env NODE_ENV=development node -r babel-register node_modules/.bin/webpack-dev-server --config ./webpack.config.development.js",
```

Comment:
Changing the path to `./node_modules/webpack-dev-server/bin/webpack-dev-server` fixed the issue I was running into on Windows.

I also think we should add `./` in-front of node_modules, so it matches the syntax of the other scripts, but it does work with it out for me.